README.md

```diff
@@ -12,70 +12,193 @@ Airport Challenge
+*Test 5* - when PlaneTakesOff is called and the weather is sunny, the length of planesAtAirport decreases by 1  
+
+
+## User Story 7  
+As an air traffic controller  
+To ensure safety  
+I want to prevent landing when weather is stormy  
+
+**Domain Model 7**  
+
+| Objects | Properties                    | Messages                          | Outputs |
+| ------- | ----------------------------  | ----------------------------      | ------- |
+| Airport |planesAtAirport @Array[@Plane] | planeLands(@Plane, @Weather)      | @Void   |
+|         |airportName@String             |                                   |         |
+|         |airportCapacity @Integer       | atCapacity()                      | @Boolean|
+|         |                               | planeTakesOff(@Plane, @Weather)   | @Void   |
+|         |                               | planeIsAtAirport(@Plane)          | @Boolean|
+| Plane   |planeID  @String               |                                   |         |
+| Weather |currentWeather @String         | checkWeather()                    |         |
+|         |possibleWeather @Array         |                                   |         |
+
+**Tests**  
+
+*Test 1* - when planeLands is called and the weather is stormy, the length of planesAtAirport remains the same and the plane doesn't land  
+*Test 2* - when planeLands is called and the weather is stormy, an error message is returned  
+*Test 3* - when planeLands is called and the weather is sunny, the length of planesAtAirport increases by 1  
+
+## User Story 8  
+As an air traffic controller  
+To count planes easily  
+Planes that have landed must be at an airport  
+
+**Domain Model 8**
+
+| Objects | Properties                    | Messages                          | Outputs |
+| ------- | ----------------------------  | ----------------------------      | ------- |
+| Airport |planesAtAirport @Array[@Plane] | planeLands(@Plane, @Weather)      | @Void   |
+|         |airportName@String             |                                   |         |
+|         |airportCapacity @Integer       | atCapacity()                      | @Boolean|
+|         |                               | planeTakesOff(@Plane, @Weather)   | @Void   |
+|         |                               | planeIsAtAirport(@Plane)          | @Boolean|
+| Plane   |planeID  @String               | planeLocation(@airport)           | @String |
+|         |location @string               |                                   |         |
+| Weather |currentWeather @String         | checkWeather()                    |         |
+|         |possibleWeather @Array         |                                   |         |
+
+**Tests**
+
+Note: I think this User Story has already been addressed and tested with the 'planeIsAtAirport' function, however I've added a function to the plane class to return if it's at a specific airport (this could be expanded to a list of airports with a for loop) or in flight  
```

Comment:
To expand, the options I considered were:
- Creating an index.js file which contained the method to call planeIsAtAirport and check the length of planesAtAirport array
- Adding a line within planeLands that calls planeIsAtAirport to return true
- The method I chose to add (planeLocation) I think I should be called within planeLands so the planes location changes, however this could get messy and is probably not the best way to achieve this (or the user story)